### DISCUSSION TITLE
關於今天的期末考

### DISCUSSION THREAD

**Discussion by Aaron0810** [1101-GS4719-C/Python-Community-GS4719-C#1488] [閒聊八卦]
今天考完這個上機期末考這堂課基本上就算是差不多結束了，不知道大家今天考的怎麼樣呢? 希望大家能來多多分享。

我自己是先寫下面的程式題最後才回去寫申論題的那些問題，我覺得這次老師出的題目其實在初級程式課程上算是偏難的，但畢竟是可以上網查資料的，所以整體算是平衡了不少，而且總分給了140算是蠻佛的。

有另一點題外話是我剛開始很認真在寫程式都沒看telegram，後來幾乎寫完了才看了一下才發現老師有丟可以直接看現在分數的那個，好險後來有發現這個讓我把錯的改了一下，不然分數可能會不太好看。

這邊也希望大家在最後這兩三個禮拜把討論區或小組討論的分數趕緊拿一拿，最後祝大家期末都歐趴。

**Comment by Superlan8** [comment]
我覺得這次考試確實對於基礎程式課程而言，後面幾題程式題對初學者來說有點困難，而且這種考試方式對於大部分同學來說都算新，蠻多同學可能不太熟悉如何運用這次的考試系統，希望老師之後再考前可以先讓同學試用，也避免像這次老師也忙到焦頭爛耳的狀況。

**Comment by Tzuhsuannnnn** [comment]
我覺得數字換英文字及九宮格轉至都不容易，需要花蠻多時間想的，但老師總分給很多，所以也可以先寫其他較容易的。我覺得整體考試下來跟期中考比起來很有趣!

**Comment by 111607004** [comment]
我也覺得這次的考試對真正的初學者來說有點難，而且還要摸索整個系統的運作，在開始作答前就花了不少時間，如果不是老師有延長我們的考試時間，我的分數可能會...痾...非常...相當慘烈QQ
不過總分140，而且申論就佔了50分，其實也已經夠友善了，老師好棒，期末救星！

**Comment by Evening218** [comment]
這次考試對於我這個初學者來說,後面幾題完全沒有頭緒最後還是上網找資料慢慢修改試出答案的,但是我覺得前面申論題有保底分數,所以其實壓力真的不大,考試的時候我就是算算分數發現大概可以了,剩下的時間就毫無壓力的在亂試程式看能不能矇到答案

**Comment by MaxChi870** [comment]
申論題超佛，程式題前面還能靠講義上的基礎方式寫出來，後面就真的得靠網上資料來輔助，但多半花時間都能完成，感謝老師的延長時間。
